src/modules/dashboard/includes/Term_Entity_Match/Term_Query.php

```diff
@@ -57,22 +57,28 @@ public function __construct( $request, $cursor, $cursor_sort, $limit ) {
 				e.about_jsonld as match_jsonld,
 				t.name,
 				t.name as term_name,
-				e.id AS match_id
+				e.id AS match_id,
+				e.match_name,
+				COUNT(DISTINCT p.ID) AS occurrences
 			FROM {$wpdb->prefix}terms t
 			INNER JOIN {$wpdb->prefix}term_taxonomy tt
-			    ON t.term_id = tt.term_id
+				ON t.term_id = tt.term_id
+			LEFT JOIN {$wpdb->prefix}term_relationships tr
+				ON tt.term_taxonomy_id = tr.term_taxonomy_id
+			LEFT JOIN {$wpdb->prefix}posts p
```

Comment:
I am still very worried at the performance of this. This how WP does it in the terms page?

Can use a faker plugin to generate 100.000+ posts with several terms relations and see what happens in terms of performance?